lib/ci-stack.ts

```diff
@@ -30,8 +30,8 @@ import { JenkinsWAF } from './security/waf';
 export interface CIStackProps extends StackProps {
   /** Should the Jenkins use https  */
   readonly useSsl?: boolean;
-  /** Should an OIDC provider be installed on Jenkins. */
-  readonly runWithOidc?: boolean;
+  /** Type of login mechanism to adopt */
+  readonly authType?:string;
```

Comment:
I see, you just completely change it to a new param.

Reply:
Yes!